### DISCUSSION TITLE
Dúvida sobre responsividade

### DISCUSSION THREAD

**Discussion by igorbrx2** [frontendbr/forum#2405] [Perguntas e Respostas]
Olá! Sou novo no front end e estou me deparando com um problema que deve ser comum: estou com uma borda lateral da cor do background em telas menores, como as de celular.

Estou enviando em anexo uma print do site em uma tela de ipad, com layout normal e outra em uma tela de smarphone, com essa borda lateral. Alguém pode me ajudar?

Normal
![Captura de tela de 2023-03-17 17-33-35](https://user-images.githubusercontent.com/124373223/226189721-eac83684-72d7-40de-84bf-1e586126feff.png)

Em tela de smartphone
![Captura de tela de 2023-03-17 17-34-04](https://user-images.githubusercontent.com/124373223/226189743-93caae90-a74d-4a07-b1b0-9d18131a3c84.png)

**Comment by ninetails** [comment]
Provavelmente algum elemento está quebrando, seja por ter um width fixo etc. Pode até ser um elemento "invisível", ou algum container que vc acabou colocando sem querer com um valor predeterminado.

Para achar o elemento responsável, tenta mandar um:

```css
* { border: solid 1px red; }
```

Aí após encontrar, arrume a responsividade do elemento em questão e tire o código acima.
